tests/test_vision_data.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from fastai import *
+from fastai.vision import *
+
+@pytest.fixture(scope="module")
+def path(request):
```

Comment:
@anishpdalal : we won't need the `request` variable anymore (not being used)

Also note that there might be a merge conflict with this PR - https://github.com/fastai/fastai/pull/1003
(as we both added the same file and the same function)

Please merge upstream and resolve the conflict if that gets merged 1st